PHP/PHPhomework3.php

```diff
@@ -0,0 +1,18 @@
+<?php
+   
+function primeNumberFinder($n)
+{
+    $x = 0;
+    $y = 1;
+    for ($i = 2; $i < $n ; $i++) {
+        $x = $n % $i;
+        $y *= $x;
+    }
+    if($y == 0) {
```

Comment:
@toru828 pls change to `===`

Reply:
Thanks for your teaching. Now, I understand what happened. Thank you very much.

Reply:
pls mark to resolve this conversation if you think we done ^^

Reply:
@toru828 seem you didn't change `==` to `===` ?